src/server/req_runjob.c

```diff
@@ -331,15 +333,22 @@ req_runjob(struct batch_request *preq)
 		return;
 	}
 	
+	if (!find_assoc_sched_jid(jid, &psched)) {
+		sprintf(log_buffer, "Unable to reach scheduler associated with job %s", jid);
+		log_err(-1, __func__, log_buffer);
+		req_reject(PBSE_IVALREQ, 0, preq);
```

Comment:
I think we should reject the request with a new error number which says corresponding scheduler not found